src/droplet.F

```diff
@@ -400,53 +416,72 @@ subroutine droplet_driver(dt,xh,uh,ruh,xf,yh,vh,rvh,yf,zh,mh,rmh,zf,mf,zs,    &
       dtpdt  = (pdata(np,prtp)-dtpdt)/dt
       dmpdt  = (rhow*4.0/3.0*pi*pdata(np,prrp)**3-dmpdt)/dt
 
+
       !Project the feedback onto the grid, using ORIGINAL location
       call project_feedback(iflag,jflag,kflag,x3d0,y3d0,z3d0,sig3d0, &
                             rhop0,taup0,rp0,tval,uten,vten,wten,qten,thten, &
                             xh,xf,yh,yf,zh,zf,sigma,sigmaf, &
                             sigdot,dvpdt1,dvpdt2,dvpdt3,drpdt,dtpdt,dmpdt, &
                             rhoval,part_grav1,part_grav2,part_grav3,pdata,np)
 
-!-----------------------------------------------------
-!  Account for boundary conditions (if necessary)
-!-----------------------------------------------------
-
-      !Now that the particles have been updated, do something with those which
-      !have fallen out the bottom:
-      if (pdata(np,pract).lt.0.0) then
-         !Replace with a new droplet at random location
-         rand = getRandomReal(randomNumbers)
-         pdata(np,prx) = rand*maxx
-         rand = getRandomReal(randomNumbers)
-         pdata(np,pry) = rand*maxy
-         rand = getRandomReal(randomNumbers)
-         pdata(np,prz) = rand*maxz
-         pdata(np,prvpx) = 0.0
-         pdata(np,prvpy) = 0.0
-         pdata(np,prvpz) = 0.0
-         !Droplet size (radius)
-         pdata(np,prrp) = 20.0e-6
-  
-         !Droplet solute mass
-         salinity = 0.034
-         pdata(np,prms) = salinity*rhow*4.0/3.0*pi*pdata(np,prrp)**3
-         !Droplet temperature
-         pdata(np,prtp) = 302.0
-         !Droplet multiplicity
-         pdata(np,prmult) = 1.0
-         !Make it alive again
-         pdata(np,pract) = 1.0
-      end if
-
     ENDDO  nploop
     !$acc end parallel
 
     !$acc end data
 
     !$acc update host(pdata,pdata_locind)
 
+    numvec(1) = num100
+    numvec(2) = num1000
+
+#ifdef MPI
+
+    if ( myid == 0 ) then
+       call mpi_reduce(mpi_in_place,numvec,2,mpi_int,mpi_sum,0,mpi_comm_world,ierr)
+    else
+       call mpi_reduce(numvec,numvec,2,mpi_int,mpi_sum,0,mpi_comm_world,ierr)
+    end if
+
+#endif
+
+    if (myid==0) write(*,*) 'num100,num1000 = ',numvec(1),numvec(2)
```

Comment:
Are these for debug purpose only or do we need to include them in a production run?

Reply:
I'll remove the print statement in the next PR, but the calculations need to happen. I need to send them to the droplet_diag